sci-physics/hepmc/hepmc-3.2.6.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+inherit fortran-2 cmake python-single-r1
+
+MYPN=HepMC3
+MYP=${MYPN}-${PV}
+
+DESCRIPTION="Event Record for Monte Carlo Generators"
+HOMEPAGE="https://hepmc.web.cern.ch/hepmc/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.cern.ch/hepmc/HepMC3"
+else
+	SRC_URI="https://hepmc.web.cern.ch/hepmc/releases/${MYP}.tar.gz"
+	S="${WORKDIR}/${MYP}"
+	KEYWORDS="~amd64"
```

Comment:
Any particular reason we are dropping the `x86` keyword?

Reply:
Oh, i think i did it again... I keep thinking I should only set the keywords for what I actually did/can test.

Reply:
Wouldn't dropping that version need stabelization of the other `:2` version (first)?

Reply:
> Wouldn't dropping that version need stabelization of the other `:2` version (first)?

Yes you are right. This package is long overdue for a stabilization request:
```
    StableRequest
    slot(2) no change in 248 days for unstable keywords: [ ~amd64, ~x86 ]
    StableRequest
    slot(3) no change in 65 days for unstable keywords: [ ~amd64, ~x86 ] 
```

Could you open one? Maybe we can also request stabilization for slot 3.

Reply:
> Oh, i think i did it again... I keep thinking I should only set the keywords for what I actually did/can test.

For major version bumps with big changes this is often a good idea. For regular version bumps it is okay to just keep the keywords that are already there. By the way, for `x86` it is also possible to test in a container on `amd64`.

Reply:
I would only consider 3.2.6 good enough for stabilization in slot 3 (cf. https://gitlab.cern.ch/hepmc/HepMC3/-/issues/59)

Reply:
> I would only consider 3.2.6 good enough for stabilization in slot 3 and that's new (cf. https://gitlab.cern.ch/hepmc/HepMC3/-/issues/59)

Alright, then lets just stabilize only slot 2 and revisit stabilization of slot 3 later.